### PR TITLE
storage node docs

### PR DESCRIPTION
**Pull request by egonelbre**

### CODE REVIEW

File: code/Storage Node.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Storage Node also uses Kademlia network to broadcast it's own status to satellites.
+
+## Piecestore
+
+Is the endpoint that implements data transfer protocol. It allows uplinks to upload and download data from the storage node.
+
+Piecestore contains logic to validate order limits, orders and piece hashes.
+
+Piecestore uses Pieces to store the data and Trust to verify allowed peers. It updates Bandwidth, Orders and other databases while uploading/downloading.
```
Comment (ifraixedes) [inline code/Storage Node.md]: It would be cool to link the _Trust_, _Bandwidth_ and _Orders_ sections.

File: code/Storage Node.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Orders system keeps track of order limits and orders and every interval settles them with the Satellite.
+
+## Bandwidth
+
+Bandwidth system keeps track of bandwidth usage as reported by order limits and orders.
+
+## Collector
+
+Collector system every interval purges expired pieces from the system.
```
Comment (ifraixedes) [inline code/Storage Node.md]: I think that it should be _of the system_.

File: code/Storage Node.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Pieces system handles writing and reading pieces from the blob store. It also provides ways to keep track of pieces meta data.
+
+## Trust
+
+Trust system contains logic for trust logic. It contains the validation logic for satellites and uplinks.
+
+## Monitor
+
+Monitor checks every interval how much free disk space there actually is and updates Kademlia as necessary.
```
Comment (JessicaGreben) [inline code/Storage Node.md]: Monitor checks how much free disk space is on the node and updates Kademlia with what?  I didn't realize Kademlia stored data about disk space, I thought it only knew other node contact info.

Comment (egonelbre) [reply code/Storage Node.md]: Yes, currently storage node broadcasts its available storage to satellites this way.

**Pull request merged**